charts/ingress-nginx/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This file documents all notable changes to [ingress-nginx](https://github.com/kubernetes/ingress-nginx) Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+### 3.35.0
+
+- [7203] https://github.com/kubernetes/ingress-nginx/pull/7203 Make HPA behavior configurable
+- [7156] https://github.com/kubernetes/ingress-nginx/pull/7156 Drop support for Ingress Object v1beta1
```

Comment:
you can drop this

Reply:
This wasn't dropped yet, please remove those lines